projects/vdk-plugins/vdk-control-api-auth/setup.py

```diff
@@ -25,14 +25,9 @@
     "authentication.",
     long_description=pathlib.Path("README.md").read_text(),
     long_description_content_type="text/markdown",
-    install_requires=[],
+    install_requires=["requests"],
     package_dir={"": "src"},
     packages=setuptools.find_namespace_packages(where="src"),
-    entry_points={
```

Comment:
looking at vdk-control-api-auth README.md, is it meant to be used by other plugins like "library";
maybe imported & explicitly instantiated? like common reusable code? is it concurrent? it is not clear by the README, so I'm just guessing; maybe provide an example snippet in the readme?

Reply:
Yes, it is a library, I'll update the README with subsequent PRs. This is only the first PR, and currently the package is not yet being released.
It is not currently concurrent, do we have a requirement for libraries to be concurrent?